### ISSUE TITLE
Use different feedin ts for pv_ground_vertical_bifacial

### ISSUE THREAD

**Issue by nesnoj** [empowerplan/epp-app#116]
Currently we use `sequences/solar-pv_ground_profile` from oemof datapackage for all PV ground. However, a different timeseries for `pv_ground_vertical_bifacial` needs to be integrated (for `pv_ground_elevated` the existing `solar-pv_ground_profile` is ok).

@henhuy 
Do we need separate components for PV ground in the esys or can we overlay both timeseries on-the-fly?

**Comment by henhuy** [comment]
Don't know if I fully understand the question.
For django-oemof it is no problem to update profiles on runtime, BUT as we want to use TSAM aggregated timeseries, timeseries have to be aggregated and stored in DP beforehand.
We could have multiple DPs prepared and select DP for simulation depending on user settings, if this helps?
Don't know how many DPs we need in this case?!

**Comment by nesnoj** [comment]
I see, so the only option seems to be adding separate pv_ground_... components incl. profiles to the esys.
Which would require adjusting the mapping between backend and sim

**Comment by henhuy** [comment]
You are right - but this would be a small change IMO (one extra line or something).
As you can see #117 I have to revisit EE calculation anyway.
But @sarah has to rerun TSAM for us as well.
